### DISCUSSION TITLE
出现图片链接错误

### DISCUSSION THREAD

**Discussion by Prokint** [liuzhihang/halo-theme-hao#790] [Q&A]
https://liuzhihang.com这个链接502错误，导致部分图片无法正常加载 如何修改这个链接指向？

**Comment by carolcoral** [comment]
> [https://liuzhihang.com这个链接502错误，导致部分图片无法正常加载](https://liuzhihang.com%E8%BF%99%E4%B8%AA%E9%93%BE%E6%8E%A5502%E9%94%99%E8%AF%AF%EF%BC%8C%E5%AF%BC%E8%87%B4%E9%83%A8%E5%88%86%E5%9B%BE%E7%89%87%E6%97%A0%E6%B3%95%E6%AD%A3%E5%B8%B8%E5%8A%A0%E8%BD%BD) 如何修改这个链接指向？

这个域名已经没用了，你在主题设置里面把涉及到相关地址全部换成自己的就行
